OpenUpSA/wazimap-ng-ui
# Adding a tooltip to the map download button when it is disabled

## Description
Adding a tooltip to the map download button when disabled telling the user to close the menu on the left to be able to download the map

## Related Issue
https://tree.taiga.io/project/jbothma-wazimap-ng/task/674

## How to test it locally
* expand any panel on the left
* hover over the map download button
* confirm that a descriptive tooltip is shown

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/131397418-baf75628-5276-4c15-b61d-a27983bb30e8.png)


## Changelog

### Added

### Updated
* `side_panels.js` to add/remove title attr to the map download button

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
